### ISSUE TITLE
Make support for windows 10

### ISSUE THREAD

**Issue by gorrioncillo** [paul-nameless/tg#201]
Please make support for windows10 at least 64bits
I try many things to run this great app, but none of them work. Maybe someone could guide me to the right direction.

I use mintty (git bash) from a portable instalation of git for windows
I create a venv `py -3.9 -m venv /path/to/venv/directory`
Then I installed your app with `pip install tg`, which install `python-telegram` lib too.

I realized that `python-telegram` doesn't come with tdjson.dll for win10 64bits so I downloaded a precompiled version from [this site](https://github.com/ForNeVeR/tdlib.native/releases).
I just give the path on `conf.py` file like `TDLIB_PATH = "/path/to/your/library/tdjson.dll"`

It seems to work when a run `tg`, ask for my login code, but as soon as I put it and press enter the program stops. If I try tu re run it the error `redirection is not supported`.
I search that on duckduckgo and it seems that is a problem of mintty console

I try cmd.exe program and the error now is `FIND: parameter format not correct`, search again and seems it is a problem of the two find programs, one from windows and one from git bash.
I just try to run on the folder where find from git bash is, but the program just do not output anything, just return to console prompt.

Maybe I am making some mistakes, if someone could lead me to the right direction, i really want to use this telegram client

**Comment by paul-nameless** [comment]
Hello, could you please check the logs, by default they should be located:
`~/.local/share/tg/`
If this path doesn't work on Windows you can try to set any other in conf.py using thig variable `LOG_PATH`

**Comment by gorrioncillo** [comment]
there is no reports of erros on the files `all.log` and `error.log`, so I think is problem of the terminal on windows 10, but I try on many other programs like conEmu

**Comment by paul-nameless** [comment]
tg uses ncurses and I don't know if it's supported on windows.
Have you tried cygwin?

**Comment by gorrioncillo** [comment]
yes, I try with python that comes with cygwinx64, but tg didn't work. I really do not know what else could it be.
I even installed windows-curses library from pypi, because there was an error when try at really first time.

**Comment by paul-nameless** [comment]
I don't know your programming skills but you could try to debug it either by setting `breakpoint()` or commenting out lines in the `main` function https://github.com/paul-nameless/tg/blob/master/tg/__main__.py
So we'll understand where is the error happen

**Comment by gorrioncillo** [comment]
I put many `print()` statements all over `main()` function, and try to run `tg` on git bash command line on windows 10. The output is the same `Redirection is not supported`. But when I run `python __main__.py` on the folder where this app is installed I see the print output.
According to those outputs the line `    wrapper(partial(run, tg))` do not run, I see that the wrapper function is from curses library.
Maybe I am using the wrong library? As I said I installed windows-curses with `pip install windows-curses`

**Comment by paul-nameless** [comment]
Wow, you did amazing work.
We could make part with signal compatible with windows.
Is there any errors in error.log which can give a hint with other problems?

**Comment by gorrioncillo** [comment]
[error.log](https://github.com/paul-nameless/tg/files/6049680/error.log)
Here is the error log. 
I notice that if I try to run `tg` from the git bash command the error `no redirection support` appear, but if I run `python /path/to/tg/__main__.py` works.
Inside tg I can see some chats, but for a lttle time, beacuse everything begins to become a mesh as soon as I try to move around with `jk`, but iit seems it is working some how because it sends messages but to "random" chat.
      Sorry for my bad english
I will try to look the error.log

**Comment by KylesDev** [comment]
Any progress on this? I have the same exact issue, can't run it on windows, i'm using a docker container in the meantime but this ends up removing the persistance of the config file and i have to log-in again every time the container is stopped
